pkg/harvester/components/settings/storage-network.vue

```diff
@@ -84,9 +84,13 @@ export default {
       const clusterNetworks = this.$store.getters[`${ inStore }/all`](HCI.CLUSTER_NETWORK) || [];
 
       return clusterNetworks.map((n) => {
+        const readyCondition = (n?.status?.conditions || []).find(c => c.type === 'ready') || {};
+        const disabled = readyCondition?.status !== 'True';
+
         return {
-          label: n.id,
+          label: disabled ? `${ n.id } (${ this.t('fleet.fleetSummary.state.notReady') })` : n.id,
```

Comment:
We should use the translation in the shell directory as little as possible, unless it is a general translation part.

Reply:
Done